doc/modules/svm.rst

```diff
@@ -679,49 +734,80 @@ Its dual is
    \textrm {subject to } & e^T (\alpha - \alpha^*) = 0\\
    & 0 \leq \alpha_i, \alpha_i^* \leq C, i=1, ..., n
 
-where :math:`e` is the vector of all ones, :math:`C > 0` is the upper bound,
+where :math:`e` is the vector of all ones,
 :math:`Q` is an :math:`n` by :math:`n` positive semidefinite matrix,
 :math:`Q_{ij} \equiv K(x_i, x_j) = \phi (x_i)^T \phi (x_j)`
 is the kernel. Here training vectors are implicitly mapped into a higher
 (maybe infinite) dimensional space by the function :math:`\phi`.
 
-The decision function is:
+The prediction is:
```

Comment:
But it is the prediction is the sign of this, isn't it?

Reply:
That's for regression (I did fix it for classification)